mslib/msui/wms_control.py

```diff
@@ -64,6 +66,20 @@ def add_wms_urls(combo_box, url_list):
         combo_box.addItem(url)
 
 
+def determine_wms_version(content):
+    """
+    Parses the version out of a GetCapabilities response
+    If parsing causes an error, return None for further investigation
+    """
+    try:
+        version = WMSCapabilitiesReader().readString(content).attrib["version"]
+        if version not in ["1.1.1", "1.3.0"]:
+            version = "1.1.1"
+        return version
+    except Exception as ex:
```

Comment:
Avoid needless wildcard exceptions. Specify the stuff you expect/know. Maybe throw in a log message as you already have ex available. I have been burnt by wildcards before.

Reply:
I removed that function entirely with the latest commit